modules/ROOT/pages/appliance/login_information.adoc

```diff
@@ -1,5 +1,5 @@
-ownCloud Appliance Login Information
-====================================
+ownCloud Appliance Login Information and Custom Paths
```

Comment:
As the document's stored inside the Appliance directory, then prefixing the header with "ownCloud Appliance" is redundant. I'd strongly suggest removing it, leaving the header as "Login Information and Custom Paths".